capnpc-go/templates/interfaceClient

```diff
@@ -26,7 +26,7 @@ func (c {{$.Node.Name}}) {{.Name|title}}(ctx {{$.G.Imports.Context}}.Context, pa
 // should not be used to compare clients.  Use IsSame to compare clients
 // for equality.
 func (c {{$.Node.Name}}) String() string {
-	return capnp.Client(c).String()
+	return {{$.G.Imports.Fmt}}.Sprintf("%T(%v)", c, capnp.Client(c))
```

Comment:
I wonder if we should actually just remove these indirections as well, like we did for `{{.G.Capnp}}`. But that doesn't need to be this pr.

Reply:
I suspect the reason for this is to avoid unneeded imports.  For instance, if your schema doesn't contain any capabilities, there's no need to import "fmt".

Reply:
Ah. I think it would be simpler to (unconditionally) just emit a block like:

```
var (
  _ fmt.Print
  ...
)
```

...for each package that might or might not be used. I think this is how the protobuf compiler does things.